modules/client-configuration/pages/client-automating-preparation.adoc

```diff
@@ -134,3 +135,20 @@ If set, local repositories will stay active (not recommended).
 
 Navigate to menu:Systems[Autoinstallation > Autoinstallation Snippets] to see which snippets are available.
 For more information, see xref:reference:systems/autoinst-snippets.adoc[].
+
+
+
+== SUSE Manager Proxy
+
+A {productname} Proxy is a SLE client with a special role.
+You can install it using AutoYaST.
+
+. Create an autoinstallation tree as outlined in <<client-cfg-autoinstallation-prep>>.
++
+* Unpack a Unified Installer DVD1.
+* Use [guimenu]``SLE-Product-SUSE-Manager-Proxy-4.1-Pool for x86_64`` as the base channel.
+* Make sure the [guimenu]``SUSE Manager Proxy 4.1 x86_64`` product is completely mirrored.
+
+. Create a suitable AutoYaST profile as outlined in xref:upgrade/proxy-migration.adoc[].
```

Comment:
Hmm, I think this reference a autoyast profile made for "migration". This is not the best thing to point to.
I did not check if it also has some text which might be usefull, but if you just need an autoyast profile for Proxy new installation, I can try to provide one.

Reply:
@mcalmer thanks for spotting!  yes, i'd appreciate if you'd provide one for a new proxy install.

Reply:
https://w3.suse.de/~mc/SUSEManager/suma-proxy-install.xml

Maybe you can add some notes to that example:

- change the root password - maybe with a reference to autoyast docs
- storage section is missing. I am not sure what happens without it. I hope the proposal is used but I cannot say for sure. A question to schubi is running. Maybe I will update it when I have more information.

Reply:
Just got feedback. If no partitioning section is present in the autoyast profile it should take the proposal. So I think this example should be ok

Reply:
Thanks Michael!  I changed the version numbers: 4.0 > 4.1 and 15-sp1 > 15-sp2.  I hope this will match the names when the product is released.